src/python/national_voter_file/us_states/mi/transformer.py

```diff
@@ -79,11 +94,21 @@ def __init__(self, input_path, *args):
     def process(self):
         if self.input_path.endswith('.zip'):
             z = zipfile.ZipFile(self.input_path)
-            return self.zip_voters(
-                os.path.join(os.path.dirname(self.input_path), 'entire_state_v.lst')
-            )
+            if not self.history:
```

Comment:
reverse this so the positive condition is first? a little bit easier to understand that way i think.